docs/docs/home/get-started.md

```diff
@@ -38,6 +38,11 @@ rill start my-rill-project
 
 The Rill web app runs locally at `http://localhost:9009` and will create code files in the `my-rill-project` directory.
 
+:::note
+Starting from v0.48, we have added the ability to self host the local Rill Developer application on a non-localhost endpoint. Please refer to the [rill start reference page](https://docs.rilldata.com/reference/cli/start) for more details on using the `--allowed-origins` flag. This method is not officially supported, please proceed at your own risk.
+:::
```

Comment:
I think we should take this out – we have *not* added the ability to self-host, it is still very much discouraged 😅  The `--allowed-origins` flag is just to provide an escape hatch for the people who do it despite the poor "support" for it.

Reply:
We can say that starting in v0.48, we've made changes to the CORS policy to only allow calls from the default HTTP URL by default for security policies. Therefore, if you are hosting Rill Developer (not officially supported), this means that you will need to pass in additional CORS origins via the `--allowed-origins` flag when starting Rill.

Reply:
made the above changes!